src/include/daos_obj_class.h

```diff
@@ -71,6 +75,9 @@ enum daos_obj_redun {
 	OR_RS_8P2,
 	OR_RS_16P1,
 	OR_RS_16P2,
+	OR_RS_4P3,
+	OR_RS_8P3,
+	OR_RS_16P3,
```

Comment:
hmm, just found that the daos_obj_id_is_ec() also need to change, please change it and run test again, thx

Reply:
Indeed